proto/process/agent.proto

```diff
@@ -1276,3 +1291,10 @@ message RouteMetadata {
 	int32 tagIndex = 2;
 	int64 tagsModified = 3;
 }
+message DomainDatabase {
+	bytes encodedDomains = 1;
```

Comment:
Instead of creating `DomainDatabase` with just one field, can we make `encodedDomains` directly a field in `message Connection`?

Reply:
I created this because it seems that this area changes, and it might be more convenient than to have to keep updating the top level connection.  I can certainly make it just one place if we want.

Reply:
Got it. I don't foresee any change being required here anytime soon, unlike the current set of changes which have been under consideration for a while. So I would vote for reducing the additional complexity.

Reply:
are there docs somewhere explaining what this type is ? `bytes` is an inscrutable type.